source/index.ts

```diff
@@ -184,9 +184,9 @@ export default class Pageres extends EventEmitter {
 	successMessage(): void {
 		const {screenshots, sizes, urls} = this.stats;
 		const words = {
-			screenshots: plur('screenshot', screenshots),
-			sizes: plur('size', sizes),
-			urls: plur('url', urls)
+			screenshots: plur('screenshot', Number(screenshots)),
```

Comment:
You're fixing this in the wrong place. `this.stats` should have the correct types.